moja-kuhinjica-frontend/src/pages/email/password/[token].tsx

```diff
@@ -8,8 +8,10 @@ import { useRouter } from 'next/router'
 import UserService from '@/service/User.service'
 import { Oval } from 'react-loader-spinner'
 import { routes } from '@/constants/constants'
+import { useTranslation } from 'react-i18next'
 
 const ResetPasswordPage = (): JSX.Element => {
+    const { t } = useTranslation()
```

Comment:
Shouldn't there be additional changes in this file that use this `t` for translation?